### PR TITLE
gext-gen-node-snippets-rest-part-2

### PR DESCRIPTION
**Pull request by joliveros**
- [x] message
  - [x] messages
  - [x] notification
  - [x] outgoing-caller-ids
  - [x] participant
  - [x] queue
  - [x] recording
  - [x] sending-messages
  - [x] short-codes
  - [x] sip
  - [x] sms
  - [x] subaccounts
  - [x] test-credentials
  - [x] token
  - [x] transcription
  - [x] usage-records
  - [x] usage-triggers
  - [x] voice

### CODE REVIEW

File: rest/message/list-get-example-1/list-get-example-1.3.x.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+// These vars are your accountSid and authToken from twilio.com/user/account
+var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var authToken = "your_auth_token";
+var client = require('twilio')(accountSid, authToken);
+
+client.messages.list(function(err, data) {
+    data.messages.forEach(function(message) {
+        console.log(message.body);
+    });
+});
```
Comment (juancarlospaco) [inline rest/message/list-get-example-1/list-get-example-1.3.x.js]: No new line at end of file X 3. *(config the editor to do it automatically)*

File: rest/message/instance-post-example-1/instance-post-example-1.3.x.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var authToken = "your_auth_token";
+var client = require('twilio')(accountSid, authToken);
+
+client.messages("MM800f449d0399ed014aae2bcc0cc2f2ec").post({
+    body: ""
+}, function(err, message) {
+    // will be an empty string
+    process.stdout.write(message.body);
+});
```
Comment (juancarlospaco) [inline rest/message/instance-post-example-1/instance-post-example-1.3.x.js]: No new line at end of file X 2.

File: rest/messages/generate-twiml-sms-voice/example-1.3.x.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  // Create TwiML response
+  const twiml = new twilio.TwimlResponse();
+  twiml.say('Hello! You will get an SMS message soon.');
+  twiml.sms('This is the ship that made the Kessel Run in fourteen parsecs?');
+
+  res.writeHead(200, {'Content-Type': 'text/xml'});
+  res.end(twiml.toString());
+}).listen(1337, '127.0.0.1');
+
+console.log('TwiML servin\' server running at http://127.0.0.1:1337/');
```
Comment (juancarlospaco) [inline rest/messages/generate-twiml-sms-voice/example-1.3.x.js]: Avoid this kind of unnecessary escape and abbreviation slang `servin\'` :arrow_right: `serving` , 
makes it easy for the people that are not keen on the English language IMHO.

Comment (joliveros) [inline rest/messages/generate-twiml-sms-voice/example-1.3.x.js]: I thought that was a bit much as well jeje

**Comment by juancarlospaco** [comment]
:+1:

**Comment by hortega** [comment]
Looks good to me, good work! :+1:

**Pull request merged**